### ISSUE TITLE
Services fail to restart automatically after RPM package upgrade

### ISSUE THREAD

**Issue by imker25** [imker25/samba_exporter#56]
After upgrading the samba-exporter package on Fedora, the `samba_exporter` service does not run. And `journalctl -u samba_exporter.service` shows the error below (or similar):

```
Feb 07 22:45:00 fedora systemd[1]: Started Prometheus samba-exporter service, based on https://github.com/imker25/samba_exporter.
Feb 07 22:45:00 fedora samba_exporter[2711]: Error: The received data "PROCESS_REQUEST: Response for request 1
Feb 07 22:45:00 fedora samba_exporter[2711]: Samba version 4.15.5
Feb 07 22:45:00 fedora samba_exporter[2711]: PID     Username     Group        Machine                                   Protocol Version  >
Feb 07 22:45:00 fedora samba_exporter[2711]: ---------------------------------------------------------------------------------------------->
Feb 07 22:45:00 fedora samba_exporter[2711]: panic: Error: The received data "PROCESS_REQUEST: Response for request 1
Feb 07 22:45:00 fedora samba_exporter[2711]: Samba version 4.15.5
Feb 07 22:45:00 fedora samba_exporter[2711]: PID     Username     Group        Machine                                   Protocol Version  >
Feb 07 22:45:00 fedora samba_exporter[2711]: ---------------------------------------------------------------------------------------------->
Feb 07 22:45:00 fedora samba_exporter[2711]: goroutine 10 [running]:
Feb 07 22:45:00 fedora samba_exporter[2711]: panic(0x55b1e82e7560, 0xc000083100)
Feb 07 22:45:00 fedora samba_exporter[2711]:         /usr/lib/golang/src/runtime/panic.go:1065 +0x565 fp=0xc00006ced8 sp=0xc00006ce10 pc=0x>
Feb 07 22:45:00 fedora samba_exporter[2711]: tobi.backfrak.de/internal/smbexporterbl/smbexporter.(*SambaExporter).Describe(0xc000135810, 0x>
Feb 07 22:45:00 fedora samba_exporter[2711]:         /builddir/build/BUILD/%{archivename}/_build/src/src/tobi.backfrak.de/internal/smbexpor>
Feb 07 22:45:00 fedora samba_exporter[2711]: github.com/prometheus/client_golang/prometheus.(*Registry).Register.func1(0x55b1e8375778, 0xc0>
Feb 07 22:45:00 fedora samba_exporter[2711]:         /usr/share/gocode/src/github.com/prometheus/client_golang/prometheus/registry.go:277 +>
Feb 07 22:45:00 fedora samba_exporter[2711]: runtime.goexit()
```

**Comment by imker25** [comment]
As workaround you can run the following commands:

```sh
sudo systemctl stop samba_exporter
sudo systemctl stop samba_statusd
sudo systemctl start samba_statusd
sudo systemctl start samba_exporter
```

**Comment by imker25** [comment]
To fix this, the `%post` section in following files should be changed like shown in the workaround:
- install/fedora/samba-exporter.from_gradle.spec
- install/fedora/samba-exporter.from_source.spec

**Comment by imker25** [comment]
Fixed with V1.14.4 release
